packages/foam-vscode/src/features/dataviz.ts

```diff
@@ -110,6 +110,15 @@ async function createGraphPanel(foam: Foam, context: vscode.ExtensionContext) {
   panel.webview.onDidReceiveMessage(
     async message => {
       switch (message.type) {
+        case "webviewStyleRequest":
+          const style = getGraphStyle();
+          console.log(style)
+          panel.webview.postMessage({
+            type: "graphStyle",
```

Comment:
it's not documented, but the naming convention for these events follows the VsCode events naming pattern.
In this case it would be something like `didUpdateStyle`

Reply:
Will update